src/PatientChecking/PatientChecking/wwwroot/css/AppointmentList.css

```diff
@@ -9,7 +9,7 @@
 }
 
 .title {
-    margin-top: 15px;
+    margin: 12px;
```

Comment:
why don't we use bootstrap instead of mannually introducing new css style?

Reply:
I have used bootstrap instead of manually css